app/_locales/fr/messages.json

```diff
@@ -11,12 +14,30 @@
   "accountName": {
     "message": "Nom du compte"
   },
+  "accountSelectionRequired": {
+    "message": "Vous devez selectionner un compte !"
```

Comment:
Could we remove the space before the exclamation point?

Reply:
In french we have a space there, it's specific to french :)